wpilibj/src/main/java/edu/wpi/first/wpilibj/Tracer.java

```diff
@@ -77,7 +77,7 @@ public void printEpochs(Consumer<String> output) {
       m_lastEpochsPrintTime = now;
       m_epochs.forEach(
           (key, value) -> {
-            sb.append(String.format("\t%s: %.6fs\n", key, value / 1.0e6));
+            sb.append(String.format("\t%s: %.6fs%n", key, value / 1.0e6));
```

Comment:
What's the rationale for using `%n` instead of `\n`?

Reply:
Ah, reading https://stackoverflow.com/questions/1883345/whats-up-with-javas-n-in-printf , it causes \r\n to be generated on Windows.  Is that what we really want in all of these cases?

Reply:
Terminal is a good question.  File wise basically all modern tools handle either in file captures.  If we're passing the string to DS.reportError() or a HAL function, the C/C++ layer is actually doing the print and will convert \n to CRLF on Windows, so this could only be a potential concern in any case on Windows if we're actually passing the output to System.out.println().